sessionctx/autocommit/autocommit.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2015 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package autocommit
+
+import (
+	"github.com/pingcap/tidb/context"
+)
+
+// Checker is the interface check if should autocommit in the context.
+type Checker interface {
```

Comment:
Any suggestion? It is already in autocommit package. So I don't want to add a "Autocommit" prefix.